packages/databricks-vscode/src/ui/bundle-resource-explorer/BundleCommands.ts

```diff
@@ -90,25 +80,27 @@ export class BundleCommands implements Disposable {
 
     @onError({popup: {prefix: "Error running resource."}})
     async deployAndRun(treeNode: BundleResourceExplorerTreeNode) {
-        if (!isRunnable(treeNode)) {
-            throw new Error(`Cannot run resource of type ${treeNode.type}`);
+        const type = treeNode.type;
+        if (!isRunnable(type)) {
+            throw new Error(`Cannot run resource of type ${type}`);
         }
         //TODO: Don't deploy if there is no diff between local and remote state
         await this.deploy();
 
         await this.bundleRunStatusManager.run(
-            treeNode.resourceKey,
-            treeNode.type
+            (treeNode as any).resourceKey,
```

Comment:
Can we avoid `any` here? Why do we not have `resourceKey` on `BundleResourceExplorerTreeNode` anymore if we still want to use it here?

Reply:
We only have it on some nodes that represent a runnable node. The nodes are being filtered out above. I will look for some workaround for this any.